packages/ts/react-crud/src/autocrud.tsx

```diff
@@ -69,6 +86,11 @@ export type AutoCrudProps<TModel extends AbstractModel = AbstractModel> = Compon
     gridProps?: AutoCrudGridProps<Value<TModel>>;
   }>;
 
+function defaultFormHeaderRenderer<TItem>(editedItem: TItem | null, disabled: boolean): JSX.Element | null | undefined {
+  const style = { color: disabled ? 'var(--lumo-disabled-text-color)' : 'var(--lumo-text-color)' };
+  return editedItem ? <h3 style={style}>Edit item</h3> : <h3 style={style}>New item</h3>;
+}
```

Comment:
Isn't it better to create the default title based on the name of the Model name e.g. Person, Address, etc., and save the `customFormHeaderRenderer` for more complex titles? IMO, something like `New Person` has a better chance of remaining as it is over `New Item`, so a higher chance of less lines of codes.

Reply:
That's a good idea, but I don't know how we would solve that currently. The only way to get the name of the model is to use the class name, which probably has the same issue as the validators in that it gets minified in production. Unless we add the name as a property somewhere this doesn't seem feasible. 

I would suggest to use the PR as is, and create an improvement ticket for using the model name, together with a hint that the generator needs to be improved to make this possible.